src/orders/dto/order.update.dto.ts

```diff
@@ -0,0 +1,12 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsString } from 'class-validator';
+import { OrderStatus } from '../enums/orders.status';
+
+export class UpdateOrderDto {
+  @ApiProperty()
+  @IsString()
+  id: string; // use string becasuse I use objectID validator in the pipe.
```

Comment:
The field `id` should not be in the request body, put this in the URL path parameter instead.